frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnField.scss

```diff
@@ -0,0 +1,22 @@
+.odc-multi-column-field {
+  &__row {
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    margin-right: var(--pf-global--spacer--lg);
+    margin-bottom: var(--pf-global--spacer--md);
+    max-height: var(--pf-global--spacer--xl);
+  }
+
+  &__col {
+    flex: 1 0 0;
+    margin-right: var(--pf-global--spacer--md);
+  }
+
+  &__col--button {
+    cursor: pointer;
+    line-height: 35px;
```

Comment:
Why 35? why not something like `2em` or some other PF4 variable?

Reply:
Updated with PF4 variable